main-app/app/index.html

```diff
@@ -11,7 +11,7 @@
 <div id="pageContainer">
     <header>
         <h1>Reece vs. Mark</h1>
-        <p>{{gameModel.winnerName}} Wins <button ng-click="findWinner()">Find Winner</button></p>
+        <p>{{gameModel.winnerName}} Wins   ::   {{gameModel.$http.promise}}::    <button ng-click="findWinner()">Find Winner</button></p>
```

Comment:
what is the $http stuff doing here - also binding to a promise makes no sense at all.
I assume this is code that went in before we spoke yesterday?